homeassistant/components/fully_kiosk/notify.py

```diff
@@ -0,0 +1,77 @@
+"""Support for Fully Kiosk Browser notifications."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from fullykiosk import FullyKioskError
+
+from homeassistant.components.notify import NotifyEntity, NotifyEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import FullyKioskDataUpdateCoordinator
+from .entity import FullyKioskEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class FullyNotifyEntityDescription(NotifyEntityDescription):
+    """Fully Kiosk Browser notify entity description."""
+
+    cmd: str
+
+
+NOTIFIERS: tuple[FullyNotifyEntityDescription, ...] = (
+    FullyNotifyEntityDescription(
+        key="overlay_message",
+        translation_key="overlay_message",
+        entity_category=EntityCategory.DIAGNOSTIC,
```

Comment:
Why make these diagnostic?

Reply:
Mostly to hide them from the automatically generated dashboard since they don't add much value having them in the UI. I can remove them if that's a bad practice.

Reply:
That's a bad practice, we should filter out notify entities instead.

Reply:
https://github.com/home-assistant/frontend/pull/21075

Reply:
Removed. Thanks for the filtering!